src/Handler/CurlFactory.php

```diff
@@ -1,28 +0,0 @@
-<?php
-
-namespace M6Web\Bundle\GuzzleHttpBundle\Handler;
-
-use GuzzleHttp\Handler\CurlFactory as GuzzleCurlFactory;
-// @TODO use something else since EasyHandle class is @internal (=> BC is not ensured)
-use GuzzleHttp\Handler\EasyHandle;
-
-/**
- * Extends the Guzzle curl factory to set curl info in response
- *
- * @deprecated the curlInfo dynamic property will be deleted
- * in favor of [the native on_stat option](https://docs.guzzlephp.org/en/latest/request-options.html#on-stats)
- */
-class CurlFactory extends GuzzleCurlFactory
-{
-    /**
-     * {@inheritdoc}
-     */
-    public function release(EasyHandle $easy): void
-    {
-        if (!is_null($easy->response)) {
-            $easy->response->curlInfo = curl_getinfo($easy->handle);
```

Comment:
this info wasn't used/displayed in the data collector (i.e. Symfony devtools)?

(just asking to make sure we didn't miss something)

Reply:
As the property doesn't exist any more , I strongly doubt so... but I can try to have a look.
Looking far into guzzlehttp/psr7 (up to 2015), it was never a property